Cargo.toml

```diff
@@ -56,7 +56,7 @@ members = [
 exclude = ["ci/nostd-check", "ci/valgrind-check"]
 
 [workspace.package]
-rust-version = "1.66.1"
+rust-version = "1.72.0"
```

Comment:
This indicates the minimal rust version supported by Zenoh (https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field). If Zenoh still supports 1.66.1 we should keep this to 1.66.1.

Reply:
Zenoh hasn't supported 1.66.1 for a long time. I checked with the cargo-msrv tool which uses binary search to find the MSRV. Although this is actually just the MSRV of the zenoh crate, other crates have a lower MSRV. But we keep one MSRV for all crates.